docs/content/ir.md

```diff
@@ -454,4 +454,4 @@ example.
 The following examples show how the test suite is used internally:
 
 * [`github.com/open-policy-agent/opa/topdown#TestRego`](https://github.com/open-policy-agent/opa/blob/main/topdown/exported_test.go)
-* [`github.com/open-policy-agent/opa/test/wasm/cmd/wasm-rego-testgen`](https://github.com/open-policy-agent/opa/blob/main/test/wasm/cmd/wasm-rego-testgen/main.go)
\ No newline at end of file
+* [`github.com/open-policy-agent/opa/test/wasm/cmd/wasm-rego-testgen`](https://github.com/open-policy-agent/opa/blob/main/test/wasm/cmd/wasm-rego-testgen/main.go)
```

Comment:
🤔 We could remove that link -- it's using a different set of test files: https://github.com/open-policy-agent/opa/tree/main/test/wasm/assets

Reply:
Not sure I follow. Do we not test the Wasm implementation against the Yaml test suite?

Reply:
Yep but in another place. These tests are different

Reply:
Alright, should we link to that location instead?

Reply:
Yep, that would be here: https://sourcegraph.com/github.com/open-policy-agent/opa@0f912fcbe2206deb3f3fe2e164f76821098cd0d0/-/blob/internal/wasm/sdk/test/e2e/external_test.go?L57

Reply:
Fixed, thanks!